src/analyses/reaching_definitions.h

```diff
@@ -128,22 +128,22 @@ class rd_range_domaint:public ai_domain_baset
   void make_top() final override
   {
     values.clear();
-    if(bv_container)
-      bv_container->clear();
+    bv_container = nullptr;
```

Comment:
As this is a bugfix: is there some regression test that shows how this is broken/now fixed?

Reply:
This issue is currently not externally observable as the `ait` framework only calls `make_bottom()` before the actual analysis when initializing the abstract states, and does not call `make_top()`.

The `bv_container` is shared by all abstract states, and should not be cleared when one of them is set to top/bottom.